src/utils/getContract.ts

```diff
@@ -0,0 +1,25 @@
+import { AddressZero } from '@ethersproject/constants'
```

Comment:
moved this into its own utils file since it was using `isAddress` and we split that into `addresses.ts` to avoid potential dep cycle.